cmd/ocsp-responder/main.go

```diff
@@ -179,8 +179,14 @@ func main() {
 		cmd.FailOnError(err, "Couldn't parse shutdown kill timeout")
 
 		m := http.StripPrefix(c.OCSPResponder.Path, cfocsp.NewResponder(source))
-
-		httpMonitor := metrics.NewHTTPMonitor(stats, m, "OCSP")
+		h := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			if r.URL.Path == "/" {
+				w.Write([]byte("hi"))
```

Comment:
Why would we need something more formal? The only thing hitting this is netcraft and no one else cares.